### DISCUSSION TITLE
DNS not working (can only use raw IP for Wireguard)

### DISCUSSION THREAD

**Discussion by ErsterImChat** [qdm12/gluetun#1377] [Q&A] [answered]
Hey,

i just set up Gluetun with a custom Wireguard Config.

Everything is working fine, but it seems i cant use a FQDN for my "VPN_ENDPOINT_IP".
When using a normal IPv4 Adress everything is okay.

The Error i get is:

`ERROR reading from environment variables: VPN provider: server selection: environment variable VPN_ENDPOINT_IP: invalid IP address: MYDYNDNSDOMAIN`

Is this known or even wanted?
If not, how can i fix it?

**Comment by bnhf** [comment, marked as answer]
@ErsterImChat 

This is actually one of the features of Gluetun. No DNS is used, or available, until after the tunnel is up-and-running. For supported providers, this means local lists are kept, and for custom providers IP addresses must be used.
